src/raven.js

```diff
@@ -1421,8 +1412,16 @@ Raven.prototype = {
     var testString = (type || '') + ': ' + (message || '');
 
     if (
-      !!this._globalOptions.ignoreErrors.test &&
-      this._globalOptions.ignoreErrors.test(testString)
+      matchFilters(
+        this._globalOptions.ignoreErrors,
+        testString,
```

Comment:
I'm not sure if we should pass this `testString` here and not create it inside `matchFilters` instead.
I know that it's done that way so that we can match both `captureException` and `captureMessage`, however we also need to cover this case – https://github.com/getsentry/raven-js/pull/1076 which with this solution won't cut it.

Reply:
The `testString` has to be created outside because `matchFilters` is generic, it's used for other pattern tests such as `includePaths`, `ignoreUrls`, `whitelistUrls`.

Also, from what I see, my solution can handle #1076 fine. `matchFilters` already accept RegExp in `ignoreErrors` array. So do you mind giving the example of how it wouldn't work?

Reply:
It'd not cover an exact match on message only test.

`/^Some Random Message$/` wouldn't work, because `testString` would be `TypeError: Some Random Message` in this scenario.